script/repack-archive.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+import os
+import subprocess
+from lib.util import scoped_cwd, make_zip, tempdir
+
+
+def GetLZMAExec(build_dir):
+    if sys.platform == 'win32':
+        lzma_exec = os.path.join(build_dir, "..", "..", "third_party",
```

Comment:
how about removing `build_dir` dependency here? something like this:
```
root_src_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), *[os.pardir] * 2))
lzma_exec = os.path.join(root_src_dir, ...)
```

Reply:
Done